### ISSUE TITLE
Missing reference.

### ISSUE THREAD

**Issue by holarissun** [apexrl/GCRL-Collection#1]
Hi,

Many thanks to the authors for their great work.

I believe the following works are also worth a mention in your work.

Policy Continuation with Hindsight Inverse Dynamics (NeurIPS'19): https://proceedings.neurips.cc/paper_files/paper/2019/file/3891b14b5d8cce2fdd8dcdb4ded28f6d-Paper.pdf 

Model-Based HER (NeurIPS'21 DRL Workshop): https://arxiv.org/pdf/2107.00306.pdf

WGCSL (ICLR'22): https://arxiv.org/pdf/2202.04478.pdf

Best,
Hao

**Comment by Ericonaldo** [comment]
Cool, you can pr and we can merge it

**Comment by holarissun** [comment]
Thanks for your prompt reply. The more important issue is this should be included in the discussion of the paper --- especially for a survey type of paper. Right?

**Comment by Ericonaldo** [comment]
Yeah but the paper has already been published in Jan 2022, so it is hard to include those papers in a revised version.
